airflow/providers/amazon/aws/hooks/logs.py

```diff
@@ -25,6 +25,13 @@
 
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
 
+# From AWS team the correct way to check for end of stream is that if the value of nextForwardToken is same
+# in subsequent calls
+# The issue with this approach is, it can take a huge amount of time (e.g. 20 seconds) to retrieve logs
+# As an intermediate solution, we decided to stop fetching logs if 3 consecutive responses are empty
```

Comment:
```suggestion
# Guidance received from the AWS team regarding the correct way to check for the end of a stream is that the value of the nextForwardToken is the same
# in subsequent calls.
# The issue with this approach is, it can take a huge amount of time (e.g. 20 seconds) to retrieve logs using this approach.
# As an intermediate solution, we decided to stop fetching logs if 3 consecutive responses are empty.
```

Some suggestions for grammar/punctuation. Feel free to use or not :) Note that the first line may now need to be shortened a bit and some of it moved down to the next line, but I didn't want to do that here and make the suggested diff look more confusing.

Reply:
I like your changes :)